tests/internal/service_name/test_imports.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+
+@pytest.mark.subprocess(env=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0"))
```

Comment:
Since the "v0" and "v1" versions are tested by all integrations, I am asserting that checking leveraged methods is enough to test this change (and ignoring functionality)